osu.Game/Screens/OnlinePlay/Multiplayer/MultiplayerLoungeSubScreen.cs

```diff
@@ -49,5 +81,30 @@ protected override void OpenNewRoom(Room room)
 
             base.OpenNewRoom(room);
         }
+
+        private class MultiplayerListingPollingComponent : ListingPollingComponent
+        {
+            private bool allowPolling;
+
+            public bool AllowPolling
+            {
+                get => allowPolling;
+                set
+                {
+                    if (allowPolling == value)
+                        return;
+
+                    allowPolling = value;
+
+                    if (!allowPolling)
+                        return;
+
+                    if (IsLoaded)
+                        PollImmediately();
+                }
+            }
+
+            protected override Task Poll() => !AllowPolling ? Task.CompletedTask : base.Poll();
```

Comment:
Can this be swapped? Reading conditionals with the condition variable swapped breaks my brain a little bit.

```suggestion
            protected override Task Poll() => AllowPolling ? base.Poll() : Task.CompletedTask;
```